openedx/core/djangoapps/zendesk_proxy/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Map urls to the relevant view handlers
+"""
+
+from django.conf.urls import url
+
+from openedx.core.djangoapps.zendesk_proxy.v0.views import ZendeskPassthroughView as v0_view
```

Comment:
The intention here is that, if/when a v1 is required, we could copy `v0/views.py` into `v1/views.py` and have some duplicated code.

Reply:
Then here we'd add `from openedx.core.djangoapps.zendesk_proxy.v1.views import ZendeskPassthroughView as v1_view`